android/src/main/java/com/googlecast/GoogleCastModule.java

```diff
@@ -51,10 +53,14 @@
     private CastSession mCastSession;
     private SessionManagerListener<CastSession> mSessionManagerListener;
 
+    private static volatile boolean isCastAvailable = true;
```

Comment:
comment:

`isCastAvailable` is volatile because `initializeCast` is called on the main thread, but
react-native modules may be initialized on any thread.